recipes/pytrilinos/build.sh

```diff
@@ -0,0 +1,49 @@
+mkdir -p build
+cd build
+
+if [ $(uname) == Darwin ]; then
+    export CXXFLAGS="-stdlib=libc++"
+fi
+
+cmake \
+  -D CMAKE_BUILD_TYPE:STRING=RELEASE \
+  -D CMAKE_INSTALL_PREFIX:PATH=$PREFIX \
+  -D BUILD_SHARED_LIBS:BOOL=ON \
+  -D TPL_ENABLE_MPI:BOOL=ON \
+  -D MPI_BASE_DIR:PATH=$PREFIX \
+  -D MPI_EXEC:FILEPATH=$PREFIX/bin/mpiexec \
```

Comment:
Also add `-D MPI_EXEC_MAX_NUMPROCS=$CPU_COUNT` to limit the number of processes that are used in travis-ci.